Maskbook/Wallet/WalletCore/WalletCoreStorage.swift

```diff
@@ -166,7 +167,9 @@ extension WalletCoreStorage {
                 try viewContext.saveOrRollback()
                 return false
             }
-            let account = Account(session: session, address: address)
+            // It coverts some abnomal address to standard. The address from metamask is all lowered case.
```

Comment:
Maybe `abnormal` and `The addresses from metamask are all in lowercase`?

Reply:
Yes. The abnormal case is that the addresses returned by metamask are all lowercase.